modules/microshift-adding-olm-to-blueprint.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-update-rpms-ostree.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-adding-olm-to-blueprint_{context}"]
+== Adding the Operator Lifecycle Manager (OLM) service to a blueprint 
+
+When you install {microshift-short}, the Operator Lifecycle Manager (OLM) package is not installed by default. You can add the `microshift-olm` package in the ostree blueprint to enable OLM in {microshift-short}.
```

Comment:
I am not sure this sentence is accurate.

`microshift-olm-4.15.0~rc.4-202401262049.p0.gd05be5f.assembly.rc.4.el9.x86_64
[redhat@dhcp-1-235-96 ~]$ rpm -ql microshift-olm`
Can you double check with the developers please ?